### PR TITLE
Port some mutation/cbm specific fields to enchantments, part 2

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Continuation of #72034
#### Describe the solution
Porting even more of strictly mutation/cbm effects to enchantment system
#### Testing
Compiled, gave some mutations in debug, see no issues

### CODE REVIEW

**Comment by Hyperseeker** [comment]
Does the carry-weight enchantment accept string format ("20 kg")? Might be better to use that instead of "20000".

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -3856,7 +3856,9 @@
     "charges_per_use": 1,
     "use_action": { "type": "transform", "msg": "The %s engages.", "target": "utility_exoskeleton_on", "active": true },
     "flags": [ "STURDY", "OVERSIZE", "BELTED", "WATER_FRIENDLY", "SLOWS_MOVEMENT", "ELECTRONIC" ],
-    "relic_data": { "passive_effects": [ { "id": "ench_exo_strength" } ] },
+    "relic_data": {
+      "passive_effects": [ { "id": "ench_exo_strength" }, { "condition": "ACTIVE", "values": [ { "value": "CARRY_WEIGHT", "add": 190000 } ] } ]
```
Comment (Hyperseeker) [inline data/json/items/tool_armor.json:3860]: This item originally had +215 kg extra carry weight capacity. The +190 kg was the other one.

Comment (GuardianDll) [reply data/json/items/tool_armor.json:3860]: Whoops, copypaste mistake

**Comment by GuardianDll** [comment]
\> Does the carry-weight enchantment accept string format
They sadly do not, and while i think it won't be hard to add it, i can't do it for myself

**Comment by GuardianDll** [comment]
waiting #72167 to fix basic build test to be merged

**Comment by RenechCDDA** [comment]
> waiting #72167 to fix basic build test that blocks this PR

#72167 should fix GCC 9, your basic build failure is unrelated.

`(all_mods)=> FOREST_STEALTH (mutation_branch) has two definitions from the same source (magiclysm)!`

**Comment by GuardianDll** [comment]
i got `completed mission 5 was not in the active_missions list` dozens of times in a row, but now it decided to change the error what the fuck

**Comment by GuardianDll** [comment]
now just wait `completed mission 5 was not in the active_missions list` to be fixed

**Comment by RenechCDDA** [comment]
Test failure seems unrelated, but is very suspiciously the first time I've seen that fail. I'd like to see the test re-run (and all builds succeed) before this gets merged.

**Pull request merged**